swagger_server/itm/itm_action_handler.py

```diff
@@ -365,6 +365,19 @@ def direct_mobile_characters(self):
         return self.times_dict[ActionTypeEnum.DIRECT_MOBILE_CHARACTERS]
 
 
+    def move_to(self, target_character: Character):
```

Comment:
we take in a `target_charcter` here but don't use it in the function.

Reply:
Yep, that is quirky.  The API takes in a target character, but with the current two-room-only implementation, it's not actually used in the code.  I can remove it from the `move_to` call if you like, but it just seemed odd to have a `move_to` call with no "direct object," and if we ever implement multiple locations, we'll need to know which character the ADM is moving to.

Or, I can add a comment explaining the above.

Reply:
Yea, I suppose we leave it because it isn't hurting anything and like you mentioned we may use it later on. Maybe we add a comment here to remind us later.

Reply:
Added comment.  Agree that it looks odd to have an unused parameter like that.